File: python/7 kyu/176.py
# https://www.codewars.com/kata/582c81d982a0a65424000201/train/python

# Passed

def arr_check(arr):
    is_every_value_an_array = all([type(val) == list for val in arr])
    return is_every_value_an_array

output = arr_check([[1], [2], [3]])
print(output)

File: nodejs/7 kyu/176.js
// https://www.codewars.com/kata/52705ed65de62b733f000064/train/javascript

// Passed

function sortList(sortBy, list) {
  let sortedArr = list.sort((obj1, obj2) => obj2[sortBy] - obj1[sortBy]);
  return sortedArr;
}

output = sortList("b", [
  { a: 2, b: 2 },
  { a: 3, b: 40 },
  { a: 1, b: 12 },
]);
console.log(output);